### PR TITLE
085_Entity > Parent: Create a new record

### PR DESCRIPTION
**Pull request by gmirzoyan**
https://trello.com/c/AocU1dNP

### CODE REVIEW

**Review by Liana-Harutyunyan — APPROVED** [review]
All tests passed

File: test/specs/example.js [hunk 1]
```diff
@@ -1,13 +1,102 @@
...
-describe('QA Project', () => {
+describe('ENTITY_PARENT: CREATE A NEW RECORD', () => {
     before(() => {
         browser.maximizeWindow();
-        browser.url('/');
-    })
-
-    it('Should verify Login page title', async () => {
-        expect(await browser.getTitle()).to.equal(expected.titleName)
+        browser.url('https://ref2.eteam.work/');
```
Comment (SamJulia) [inline test/specs/example.js:6]: don't need to add url here

File: test/specs/example.js [hunk 2]
```diff
@@ -1,13 +1,102 @@
...
+            expect(recordArr[i].getText()).to.equal(inputData[i]);
+        }
+    });
+    it('20/EP-Should verify showing only one row in result', () => {
+        const resultLine = $(".float-left.pagination-detail");
+        expect(resultLine.getText()).to.equal('Showing 1 to 1 of 1 rows');
     });
-});
\ No newline at end of file
+});
```
Comment (SamJulia) [inline test/specs/example.js:102]: very good 👌🏻 need to create a new page for this and try to have an assertion for each it

**Review by SamJulia — CHANGES_REQUESTED** [review]

File: test/specs/example.js (hunk 2 above)
Comment (gmirzoyan) [reply test/specs/example.js:102]: Thank you for review. I will make requested changes in the new PR.

**Pull request closed without merging**